apps/test/unit/code-studio/editorReduxTest.js

```diff
@@ -70,29 +36,6 @@ describe('editorRedux reducer tests', () => {
   let initialState;
   beforeEach(() => (initialState = getInitialState()));
 
-  it('reorder levels', () => {
```

Comment:
is there somewhere in the lesson editor this test case could be added?

Reply:
See https://github.com/code-dot-org/code-dot-org/pull/36625